maligned.go

```diff
@@ -22,10 +23,19 @@ var fset = token.NewFileSet()
 func main() {
 	flag.Parse()
 
+	importPathPatterns := flag.Args()
+	if len(importPathPatterns) == 0 {
+		importPathPatterns = []string{"."}
```

Comment:
I think this isn't necessary - my tool doesn't do it. IIRC gotool already uses `[]string{"."}` if the list is empty.